tests/WebAppCallsWebApiCallsGraph/TodoListService/Startup.cs

```diff
@@ -67,6 +68,22 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             services.AddSingleton<ILongRunningProcessContextFactory, LongRunningProcessContextFactory>();
 
+            services.AddAuthorization();
+            //services.AddAuthorization(options =>
```

Comment:
Let's keep this code until we write the documentation of all the possibilities:
- extension method on policy builder
- requirement
- attribute.